src/apps/Hosting/Files.js

```diff
@@ -89,7 +89,7 @@ const HostingFilesView = React.createClass({
         <InnerToolbar title="Website Hosting">
           <Show if={items.length && !isLoading}>
             <RaisedButton
-              label={isDefaultHosting ? 'Published' : 'Publish'}
+              label={isDefaultHosting ? 'Default' : 'Set as default'}
```

Comment:
label should be more descriptive, eg. `Set this Hosting as default`, it's quite long, but just `Set as default` is too little in my opinion